Variables/Undefined variable/tests/test_task.py

```diff
@@ -0,0 +1,35 @@
+import importlib
+from io import StringIO
+import sys
+import unittest
+from unittest.mock import patch
+
+
+def try_import():
+    import undefined_variable
+    return undefined_variable
+
+
+class TestCase(unittest.TestCase):
+    task_name = 'undefined_variable'
+
+    def setUp(self):
+        try:
+            # Stores output from print() in actualOutput
+            with patch('sys.stdout', new=StringIO()) as self.actualOutput:
+                # Loads submission on first test, reloads on subsequent tests
+                if self.task_name in sys.modules:
+                    importlib.reload(sys.modules[self.task_name])
+                else:
+                    importlib.import_module(self.task_name)
+        except NameError as ne:
+            pass
```

Comment:
Maybe we could show the student the actual error that occurs? Right now we say 'Check what happens if you use a variable which is not defined yet' and if you do that - nothing happens

Reply:
Unfortunately, I am not sure we can at the moment. I'll add an instruction in the description to lunch the code in the editor before checking - this is the only way now.